tools/scripts/ansible/roles.yml

```diff
@@ -1,3 +1,9 @@
+- src: robertdebock.bootstrap
+  version: 4.6.2
```

Comment:
the version numbers are probably wrong except for tomcat; again I'm not sure whether bootstrap/etc are needed here